src/Core/AdminConsole/OrganizationFeatures/OrganizationDomains/CreateOrganizationDomainCommand.cs

```diff
@@ -59,7 +59,7 @@ public async Task<OrganizationDomain> CreateAsync(OrganizationDomain organizatio
         }
         catch (Exception e)
         {
-            _logger.LogError("Error verifying Organization domain.", e);
```

Comment:
ℹ️ I am surprised these worked previously, or maybe the exception was just suppressed. Also caught by `dotnet format`.